puterbot/strategies/naive.py

```diff
@@ -1,26 +1,23 @@
-"""
-Implements a naive playback strategy wherein the InputEvents are replayed
-directly, without considering any screenshots.
-"""
-
 from pprint import pformat
 import time
 
 from loguru import logger
 import mss.base
 
+
 from puterbot.events import get_events
 from puterbot.utils import display_event, rows2dicts
 from puterbot.models import Recording
 from puterbot.strategies.base import BaseReplayStrategy
+from puterbot.strategies.ocr_mixin import OCRReplayMixin
 
 
 DISPLAY_EVENTS = False
 REPLAY_EVENTS = True
 SLEEP = True
 
 
-class NaiveReplayStrategy(BaseReplayStrategy):
+class NaiveReplayStrategy(OCRReplayMixin, BaseReplayStrategy):
```

Comment:
Why modify NaiveReplay?

Reply:
It was from another issue I was working on I thought I reset the changes but apparently not.